src/vivarium_census_prl_synth_pop/components/household.py

```diff
@@ -107,12 +135,15 @@ def create_households(
             num_households, taken=self._households["address_id"]
         )
         po_boxes = self.generate_po_boxes(num_households)
+        states_pumas = self.generate_states_pumas(household_ids)
```

Comment:
This doesn't seem right because it throws away information about where the ACS-sampled households actually were! I think `states_pumas` should be an argument to the `create_households` method.

And likewise, `generate_states_pumas` should be called `sample_random_states_pumas` or something to that effect.

Reply:
Perhaps you could still do this random sampling when the `states_pumas` argument is `None`, to streamline the new-household-migration case.

Reply:
Oof, yeah, this is a big mistake! I'll work on it